.github/workflows/build.yml

```diff
@@ -38,6 +38,16 @@ on:
         default: addon
         required: false
         type: string
+      napi-rs:
+        description: Whether or not this build is for a napi-rs project.
+        default: false
+        required: false
+        type: string
+      directory-path:
```

Comment:
Used in the case that the napi-rs project is located in a subdirectory relative to the repository root. This is the case for our serverless agentless work, which lives in a subdirectory in libdatadog.